website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,126 @@
+---
+
+## Getting started as a TAG Environmental Sustainability Contributor
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-11-getting-started-as-a-tag-environmental-sustainability-contributor/tag-env-logo.png" alt="TAG Environmental Sustainability Logo"></p>
+
+If you are reading this blog it might be that you are looking forward to starting contributing to sustainability in the cloud native world.  This document will help you, and you can start by becoming a member of the TAG or one of its associated working groups.
+
+Starting to contribute to open source and cloud native communities may often seem challenging and scary for beginners, but there are many ways you can embark on this journey. One way to join the community is to become a member of one of the Cloud Native Computing Foundation Technical Advisory Groups (CNCF TAG).
+
+The Technical Advisory Group consists of members that focus on a specific area of the CNCF landscape by providing guidance, addressing challenges from the community and establishing domain-specific standards and best practices.
+
+CNCF Technical Advisory Group Environmental Sustainability is always open for new members and we hope that this article can help you clarify any concerns and questions you may have around joining the TAG, as well as make it easier for you to take those first steps in joining the community.
+
+With this blog post we would like to cover a few topics that you may be wondering about when it comes to getting started as a TAG ENV Contributor:
+
+- What TAG Environmental Sustainability (TAG ENV) is all about, what its mission, goals and non-goals are, as well as what kind of activities and subgroups exist in the TAG.
+- Why YOU joining the TAG Environmental Sustainability makes an important and positive impact in the cloud native sustainability space and what YOU can gain from joining us.
+- Step-by-step walkthrough of different ways for you to join the TAG and get involved in our ongoing initiatives.
+- How YOU can share ideas, discussion topics, improvements and any other suggestions to the TAG ENV. It’s never enough to say that all feedback is highly appreciated and welcomed in the group!
+
+## Introducing Technical Advisory Group Environmental Sustainability (TAG ENV)
+
+The Technical Advisory Group Environmental Sustainability, commonly referred to as TAG ENV, operates as part of the Cloud Native Computing Foundation (CNCF) alongside several other [TAGs](https://github.com/cncf/toc/tree/main/tags#current-tags).
+
+Drawing inspiration from international frameworks such as the Paris Climate Accords, the TAG ENV sets out with a clear agenda: contribute to tackling global environmental challenges by identifying ways to reduce the environmental footprint of steadily growing data centers and expanding cloud native technologies landscape.
+
+TAG ENV's mission centers around advocating for, developing, and supporting environmental sustainability initiatives specific to cloud native technologies.
```

Comment:
```suggestion
```
There is a section below that goes into more detail. IMO, this sentence reads a bit like a repetition of the previous section.